pkg/sdn/plugin/vnids.go

```diff
@@ -302,6 +312,14 @@ func (node *OsdnNode) updatePodNetwork(namespace string, netID uint32) error {
 			errList = append(errList, err)
 		}
 	}
+
+	// Update namespace references in egress firewall rules
+	for _, policy := range policies {
+		if err = node.UpdateEgressNetworkPolicy(policy, netID); err != nil {
```

Comment:
Do we need to delete egress related ovs rules with old-netid...DeleteFlows("table=9, reg0=%d", Old-vnid) or it's already done by UpdatePod()?